app/javascript/forms/provider-forms/cloud-provider-form/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+
+import { API } from '../../../http_api';
+import MiqFormRenderer from '../../data-driven-form';
+
+const loadProviderServerZones = () =>
+  API.get('/api/zones?expand=resources&attributes=id,name,visible&filter[]=visible!=false&sort_by=name')
+    .then(({ resources }) => resources.map(({ name, id }) => ({ value: id, label: name })));
+
+const initialSchema = emsTypes => ({
+  fields: [{
+    component: componentTypes.TEXT_FIELD,
+    name: 'name',
+    label: __('Name'),
+    options: emsTypes,
+  }, {
+    component: componentTypes.SELECT,
+    name: 'type',
+    label: __('Type'),
+    placeholder: `<${__('Choose')}>`,
+    options: emsTypes,
+    validate: [{
+      type: validatorTypes.REQUIRED,
+    }],
+  }, {
+    component: componentTypes.SELECT,
+    name: 'zone',
+    label: __('Zone'),
+    placeholder: `<${__('Choose')}>`,
+    loadOptions: loadProviderServerZones,
+    initialValue: '1',
```

Comment:
Maybe:
```suggestion
    initialValue: 'default',
```
but you might have issues if the `default` zone doesn't exist. I don't know, depends on the implementation of DDF.